src/NuGet.Core/NuGet.Protocol/RemoteRepositories/RemoteV2FindPackageByIdResource.cs

```diff
@@ -7,7 +7,10 @@
 using System.Globalization;
 using System.IO;
 using System.Linq;
+using System.Net;
+using System.Net.Http;
 using System.Net.Http.Headers;
+using System.Net.Sockets;
```

Comment:
Remove `System.Net` and `System.Net.Sockets`.  They're unused.

Reply:
fixed in next iteration.